README.md

```diff
@@ -1050,6 +1050,18 @@ Kind: *Indication*
 Errors: 
 - `BadOrigin`
 
+### `SetAssetClaimer`
+
+Set a claimer to the dropped assets.
```

Comment:
Is the term "dropped assets" used anywhere else in this document? If not, then I suggest simply calling it "assets in holding after execution finishes".

Reply:
Also, explaining the "claimer" term is also a good piece of text to have.

Reply:
Removed mention of the "dropped assets". Added the definition of a claimer to the description of the new instruction.